component/main.jsonnet

```
@@ -12,9 +12,7 @@ local namespace = operatorlib.validateInstance(params.namespace);
   [namespace]: [
     kube.Namespace(namespace) {
       metadata+: {
-        annotations+: {
-          'openshift.io/node-selector': '',
-        },
+        annotations+: params.namespaceAnnotations,
```

We should allow users to also remove annotations in the hierarchy:

```suggestion
        annotations+: std.prune(params.namespaceAnnotations),
```